pyproject.toml

```diff
@@ -103,6 +103,7 @@ ignore_errors = true
 [[tool.mypy.overrides]]
 module = [
     'arelle.plugin.validate.ESEF.*',
+    'arelle.plugin.validate.ESEF_2022.*',
```

Comment:
Hi @gmongelli. If you remove this line we can merge this PR. We're happy to add type hints and get type checking working for ESEF 2022 after this initial PR is merged.